prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/git/GitCloneTask.java

```diff
@@ -44,12 +43,12 @@ public WorkReport execute(WorkContext workContext) {
 		String gitBranch = null;
 
 		try {
-			gitUri = WorkContextDelegate.getRequiredValueFromRequestParams(workContext, GitUtils.getUri());
-			gitBranch = WorkContextDelegate.getOptionalValueFromRequestParams(workContext, GitUtils.getBranch(),
-					"main");
+			gitUri = this.getRequiredParameterValue(workContext, GitUtils.getUri());
+			gitBranch = this.getOptionalParameterValue(workContext, GitUtils.getBranch(), "main");
```

Comment:
Please add "main" in a static variable. 
Since all params are defined in GitUtils, it could be stg like ?
```
@Getter
static final String defaultBranch = "main"
```